scripts/mainsite.js

```diff
@@ -0,0 +1,213 @@
+var lastStreamer, existingIframe;

```

Comment:
nit: let's be consistent with let vs var. Because we're likely targeting users who have Chrome/Firefox, (fuck IE lol) just use `let`. `var` has too many inconsistencies with variable scope.